packages/client/src/StreamrClient.ts

```diff
@@ -568,7 +568,7 @@ export class StreamrClient {
      * Used to set known entry points for a stream partition. If entry points are not set they 
      * will be automatically discovered from the Streamr Network.
     */
-    async setStreamPartEntryPoints(streamDefinition: StreamDefinition, entryPoints: JsonPeerDescriptor[]): Promise<void> {
+    async setStreamEntryPoints(streamDefinition: StreamDefinition, entryPoints: JsonPeerDescriptor[]): Promise<void> {
```

Comment:
Was the name of the method intentionally reverted in commit 9fcd5e8a084cff65a2aaf0682e20b4ef85e6d565? As we discussed, this is an advanced feature which is allowed to use streamParts? As it does so, it is good that the fact is in the method name. Maybe could also change the first parameter to array of `StreamPartID` or `streamId + definition`?

@harbu Do you think we should or should not expose the `StreamPartID` internal type in the API in these cases? E.g. `Stream#getStreamParts` already uses the `StreamPartID` type. (not sure whether it should do that)